src/vec_bcd_ppc.h

```diff
@@ -147,55 +155,39 @@ vec_bcdadd (vui32_t a, vui32_t b)
             : "v" (a),
               "v" (b)
             : "cr6" );
-#else
-#if 0
```

Comment:
curious what's going on here. is this also a temporary disable pending infrastructure?

Reply:
My comment here was not asking about cr6, but why are you hiding the code below `#if 0`?